armory/scenarios/configs/librispeech_dev_clean_baseline_raw_sincnet.json

```diff
@@ -23,18 +20,24 @@
     },
     "defense": null,
     "evaluation": {
-        "eval_file": "armory.scenarios.raw_audio_scenario"
+        "eval_file": "armory.scenarios.base"
     },
     "metric": null,
     "model": {
-        "model_kwargs": {
-            "model_file": "model_sincnet.pth",
-            "pretrained": true
+        "fit_kwargs": {
+            "nb_epochs": 20000
         },
+        "model_kwargs": {},
         "module": "armory.baseline_models.pytorch.sincnet",
         "name": "get_art_model",
+        "weights_file": "SincNet/model_sincnet.pth",
```

Comment:
Will need an update on S3. Do we want this to be `pytorch/sincnet/model_sincnet`? Or prefer we work out the directory structure once they're all merged?

Reply:
Good catch - I'll update the structure so that it works with the present setup, and we should work out the directory structure all at once.

Reply:
Should work now.